REQUIRE

```diff
@@ -0,0 +1,2 @@
+julia 0.3
+Compat
```

Comment:
if the choice is between being able to install a version with deprecation bounds vs Pkg.add giving an error because of too-tight version bounds, I'll take the warning